library-best-practices/JLBP-4.md

```diff
@@ -2,14 +2,14 @@
 --------------------------------------------------------------
 
 - Unstable libraries are libraries that allow breaking changes to their
-  public API within the same major version. For libraries following semantic
+  public APIs within the same major version. For libraries following semantic
```

Comment:
API singular, I think

Also, "Unstable libraries are libraries that allow" can be shortened to "Unstable libraries allow"

Reply:
I think plural is correct when there are multiple of them, to match "their" in this case.

> Google Cloud APIs are a key part of Google Cloud Platform,
https://cloud.google.com/apis/docs/overview?hl=en

Updated > Unstable libraries allow